tetris.sh

```diff
@@ -465,10 +526,11 @@ stty_g=$(stty -g)              # let's save terminal state ...
 
 at_exit() {
     kill $ticker_pid                             # let's kill ticker process ...
-    xyprint $GAMEOVER_X $GAMEOVER_Y "Game over!"
+    xyprint $GAMEOVER_X $GAMEOVER_Y "$i18n_game_over"
     echo -e "$screen_buffer"                     # ... print final message ...
     show_cursor
     stty $stty_g                                 # ... and restore terminal state
+    read -rsn1                                   # ... press any key to exit ...
```

Comment:
Hmmm, why do you think this may be needed?

Reply:
because, cool-retro-term does not parse multiple arguments or command, when program stopped cant' show any message.

for this reason i put "read" command to script.

I found a solution for this problem, now I can remove it.